_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -448,13 +448,16 @@ public function get_all_options() {
 					break;
 
 				case 'onboarding':
+					$business_address = get_option( 'jpo_business_address' );
+					$business_address = $business_address ? array_map( array( $this, 'decode_special_characters' ), $business_address ) : $business_address;
```

Comment:
Since this is PHP, is the `$business_address ?` criterion enough? Or do we need something like `! is_empty()` or even `is_array()` in order to guarantee that `array_map()` isn't going to fail?

Reply:
👍 - fa7706da7